src/session/session_dhandle.c

```diff
@@ -218,11 +218,11 @@ __wt_session_lock_dhandle(WT_SESSION_IMPL *session, uint32_t flags, bool *is_dea
 }
 
 /*
- * __wt_session_release_dhandle --
+ * __wt_session_release_dhandle_gen --
```

Comment:
What does the suffix `_gen` mean? To me, and usually in WT, it means working with generation numbers and that is not the case here. Can you use a different suffix? Maybe `_int` for internal?

Reply:
Done. I am changed the name to __wt_session_release_dhandle_v2(). Never-the-less the answer to your question follows.

It means general. It is a generalization of __wt_session_release_dhandle() which always has check_visibility=false since __wt_session_release_dhandle_gen() has check_visibility as a parameter. I introduced it since C does not have default parameters. A similar thing is done in OS system calls, Windows does this, where a newer version has Ex added for extended. I thought I remembered _gen used for this but I might have remembered Ex.

Reply:
To make it easier to see what is left I am going to resolve some conversations.